platform/core/PolyPodCoreAndroid/core/src/main/java/coop/polypoly/core/Core.kt

```diff
@@ -0,0 +1,66 @@
+package coop.polypoly.core
+
+import Failure
+import FeatureManifest
+import java.nio.ByteBuffer
+
+@ExperimentalUnsignedTypes
+class Core {
+    companion object {
+        fun bootstrapCore(languageCode: String): Result<Unit> {
```

Comment:
For the record, I'm not a huge fan of having to deal with two different error handling mechanisms - in the Android app we use exceptions consistently, now in the core Android bindings we're throwing around result objects. I think it's OK-ish because it's a separate module at the end of the day, but felt like pointing it out anyway :)

Reply:
Agree, I missed that detail - that PolyPodAndroid use Exceptions instead. Better we keep it consistent, I will update the code :)